funnel/templates/email_project_registration.html.jinja2

```diff
@@ -0,0 +1,12 @@
+{% extends "email_layout.html.jinja2" %}
+{% block content -%}
+
+<p>{% trans user=user.fullname %}Hi {{ user }},{% endtrans %}</p>
+
+<p>{% trans project=project.title %}You have successfully registered for <b>{{ project }}</b>.{% endtrans %}</p>
+
+<p>{% trans next_session=project.next_session_at|datetime %}The next session in the schedule will begin at {{ next_session }}{% endtrans %}</p>
+
+<p>{% trans project_url=project.url_for(_external=True) %}More details about the project are available at this link: {{ project_url }}{% endtrans %}</p>
```

Comment:
`<p><a class="btn" href="{{ project.url_for(_external=true) }}">{% trans %}Project page{% endtrans %}</a></p>`

(Jinja2 uses lowercase `true`, `false` and `none`.)